volume/store/db.go

```diff
@@ -0,0 +1,79 @@
+package store
+
+import (
+	"encoding/json"
+
+	"github.com/boltdb/bolt"
+	"github.com/pkg/errors"
+)
+
+var volumeBucketName = []byte("volumes")
+
+type dbEntry struct {
+	Key   []byte
```

Comment:
It's interesting that a volume name is both the `Key` and part of the `Value` (which is `volumeMetadata`).